packages/swing-store-lmdb/src/lmdbSwingStore.js

```diff
@@ -176,6 +195,7 @@ function makeLMDBSwingStore(dirPath, forceReset, options) {
       txn.commit();
       txn = null;
     }
+    snapStore.commitDeletes();
```

Comment:
Let's add a comment here to remind our future selves of the important sequencing requirement: the kvstore (which used to contain `vatA -> snapshot1`, and is being replaced with `vatA -> snapshot2`) must be committed before we delete `snapshot1`. Otherwise, on restart, we'll consult the kvstore and see `snapshot1`, but we'll fail to load it because it's been deleted already.